php-frontend/pom.xml

```diff
@@ -58,6 +58,12 @@
       <groupId>org.sonarsource.analyzer-commons</groupId>
       <artifactId>sonar-regex-parsing</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.sonarsource.sonarqube</groupId>
+      <artifactId>sonar-plugin-api-impl</artifactId>
+      <version>${version.sonar}</version>
+      <scope>test</scope>
```

Comment:
I would move those lines to the root `pom.xml` of the project inside `<dependencyManagement>`.
Same thing for [sonar-php-plugin/pom.xml](https://github.com/SonarSource/sonar-php/pull/968/files#diff-38c209406b4c179e216291ae4a6a08cb573d2807fd096b37eb60baace3722206)